lunatik.lua

```diff
@@ -52,6 +52,9 @@ driver.__runtimes = {}
 driver.__threads = {}
 
 function driver:run(script, ...)
+	if self.__runtimes[script] then
+		return string.format ("%s is already running", script)
```

Comment:
you're still using spaces between function name and parenthesis; please mind it as whole in your patches (not only in the indicated lines).

what's the result of this line? is it printing the error on the command line (or in dmesg)?

Reply:
Updated it. The messages is printed on the command line, not in dmesg.